templates/etc_nginx_sites_storage-ssl-gunicorn.conf.j2

```diff
@@ -26,23 +26,13 @@ server {
     # Adjust to taste
     client_max_body_size 256M;
 
-    location /static {
```

Comment:
We've started using WhiteNoise in #217 so we don't need to serve static files from Nginx anymore.